File: cybv.c
#include <stdio.h>

int main(void)
{
    int t;
    scanf("%i", &t);

    for(int i=0; i < t; i++) {
        int n, k;
        scanf("%i", &n);
        scanf("%i", &k);

        int arr[n];
        for(int j = 0; j < n; j++) {
            arr[j] = 0;
        }

        while(k > 0) {
            for(int l = 0; l < n; l++) {
                arr[l]++;
                k--;
                if(k<=0) {
                    break;
                }
            }
        }
        int min = arr[0];
        for(int q = 0; q < n; q++) {
            if(arr[q] < min) {
                min = arr[q];
            }
        }
        printf("%i\n", min);
    }
}

File: cybv.py
t = int(input())

for i in range(t):
    n, k = input().split()
    n, k = int(n), int(k)

    arr = []

    for j in range(n):
        arr.append(int(0))
    
    while k > 0:
        for l in range(n):
            arr[l] += 1
            k -= 1
            if k <= 0:
                break
    # for w in range(n):
        # print(arr[w], end=', ')
    
    min = arr[0]
    for q in range(n):
        if arr[q] < min:
            min = arr[q]
    print(min)